src/app/warren/settings.py

```diff
@@ -238,8 +238,16 @@ class Base(Configuration):
     }
 
     SIMPLE_JWT = {
-        "ACCESS_TOKEN_LIFETIME": timedelta(minutes=5),
-        "REFRESH_TOKEN_LIFETIME": timedelta(days=1),
+        "ACCESS_TOKEN_LIFETIME": timedelta(
+            seconds=values.FloatValue(
+                default=300, environ_name="WARREN_APP_ACCESS_TOKEN_LIFETIME"
```

Comment:
I think you should add `environ_prefix=None` or else, it will expect an environment variable called `WARREN_WARREN_APP_ACCESS_TOKEN_LIFETIME`:

```suggestion
                default=300, environ_name="WARREN_APP_ACCESS_TOKEN_LIFETIME", environ_prefix=None
```